docs/docsite/rst/scenario_guides/guide_vmware.rst

```diff
@@ -0,0 +1,30 @@
+.. _vmware_ansible:
+
+********************
+VMware Ansible Guide
+********************
+
+Welcome to the Ansible for VMware Guide!
+
+The purpose of this guide is to teach you everything you need to know about using Ansible with VMware.
+
+To get started, please select one of the following topics.
+
+.. toctree::
+   :maxdepth: 1
+
+   vmware_scenarios/vmware_intro
```

Comment:
Is this a case where `vmware_scenarios/*` would work, or are you wanting to define a specific order?

Reply:
needs to be a specific order imo...otherwise vmware_concepts would come before vmware_intro, wouldn't it?

Reply:
Yep, I was going for a specific order. We could use a `blob` TOC with `vmware_scenarios/*` if we renamed the scenarios to start with numbers: `1_intro`, then `2_next thing`, etc., but I think that kind of ruins the usefulness of the `blob` approach - contributors would have to understand and follow the naming convention for that to work.